app/sequencescape_excel/sequencescape_excel/worksheet/data_worksheet.rb

```diff
@@ -1,118 +0,0 @@
-# frozen_string_literal: true
-
-module SequencescapeExcel
```

Comment:
Was this not needed as the one in the sample_manifest_excel folder is used instead?